etcdctlv3/README.md

```diff
@@ -323,3 +323,19 @@ Simple reply
 
 [etcdrpc]: ../etcdserver/etcdserverpb/rpc.proto
 [storagerpc]: ../storage/storagepb/kv.proto
+
+## Compatibility Support
+
+etcdctl is still in its early stage. We try out best to ensure fully compatible releases, however we might break compatibility to fix bugs or improve commands. If we intend to release a version of etcdctl with backward incompatibilities, we will notice users before releases and talk about how to upgrade smoothly.
+
+### Input Compatibility
+
+Input includes command name, its flags and arguments. We ensure compatibility of the input of normal commands in non-interactive mode.
```

Comment:
includes the command name, its flags, and its arguments. 

We ensure backwards compatibility?